doc-assets/node/18005-search-videos/18005.html

```diff
@@ -293,53 +189,74 @@ <h3>Notes</h3>
 
     <p>You can mark a search term as required (returned videos MUST match) or excluded (returned videos must NOT match). This is controlled with a URI-encoded <code>+ (%2B)</code> or <code>-</code> sign immediately preceding the term.</p>
 
-    <p>Note that because of the potential confusion between "+" as an indicator of a space or a required term, you must encode it as <code>%2B</code> when using it to indicate a required term.</p>
+    <p>You must encode <code>+</code> it as <code>%2B</code> when using it to indicate a required term.</p>
 
     <table class="bcls-table">
       <caption class="bcls-caption--table">Required/Excluded Terms</caption>
       <thead class="bcls-table__head">
-        <tr>
-          <th>example</th>
-          <th>url-encoded</th>
-          <th>meaning</th>
-        </tr>
+      <tr>
+        <th>example</th>
+        <th>url-encoded</th>
+        <th>meaning</th>
+      </tr>
       </thead>
       <tbody class="bcls-table__body">
-        <tr>
-          <td><code>+foo</code></td>
-          <td><code>%2Bfoo</code></td>
-          <td>videos MUST include the term <code>foo</code> in the <code>name</code>, <code>description</code>, <code>long_description</code>, <code>tags</code>, <code>reference_id</code> or <code>custom_fields</code></td>
-        </tr>
-        <tr>
-          <td><code>+custom_fields:foo</code></td>
-          <td><code>%2Bcustom_fields:foo</code></td>
-          <td>video MUST include the value <code>foo</code> for some custom field</td>
-        </tr>
-        <tr>
-          <td><code>-foo</code></td>
-          <td><code>-foo</code></td>
-          <td>videos must NOT include the term <code>foo</code> in the name, <code>description</code>, <code>long_description</code>, <code>tags</code>, <code>reference_id</code> or <code>custom_fields</code></td>
-        </tr>
-        <tr>
-          <td><code>-name:foo</code></td>
-          <td><code>-name:foo</code></td>
-          <td>videos must NOT include the term <code>foo</code> in the <code>name</code></td>
-        </tr>
+      <tr>
+        <td><code>+foo</code></td>
+        <td><code>%2Bfoo</code></td>
+        <td>videos MUST include the term <code>foo</code> in the <code>name</code>, <code>description</code>, <code>long_description</code>, <code>tags</code>, <code>reference_id</code> or <code>custom_fields</code></td>
+      </tr>
+      <tr>
+        <td><code>+custom_fields:foo</code></td>
+        <td><code>%2Bcustom_fields:foo</code></td>
+        <td>video MUST include the value <code>foo</code> for some custom field</td>
+      </tr>
+      <tr>
+        <td><code>+foo -bar</code></td>
+        <td><code>%2Bfoo%20-bar</code></td>
+        <td>videos MUST include the term <code>foo</code> but must NOT include the term <code>bar</code> in the <code>name</code>, <code>description</code>, <code>long_description</code>, <code>tags</code>, <code>reference_id</code> or <code>custom_fields</code></td>
```

Comment:
Examples no longer use exclusion as the primary search criteria.  Now focusing on using it as a filter in a more complex query